Source/player.h

```diff
@@ -288,14 +288,23 @@ struct Player {
 	 * @brief Contains Data (Sprites) for the different Animations
 	 */
 	std::array<PlayerAnimationData, enum_size<player_graphic>::value> AnimationData;
+	/* Nothing Frames */

```

Comment:
nothing frames? ...

Reply:
Yes., hence NFrames

Reply:
aaand what do they do?

Reply:
They're when the hero is doing nothing :pukaczu:

Reply:
that's the shittiest name ever, that's idle frames :S

Reply:
Neutral